libs/designsystem/src/lib/components/card/card-header/card-header.component.ts

```diff
@@ -9,8 +9,15 @@ export class CardHeaderComponent implements OnInit {
   @Input() title: string;
   @Input() subtitle: string;
   @Input() isTitleBold: boolean;
+  @Input() flagged: 'success' | 'warning' | 'danger' | 'informational' = null;
 
   constructor() {}
 
   ngOnInit() {}
+
+  @HostBinding('class')
+  get _cssClass() {
+    // TODO: Should we use the 'flagged-' prefix or not?
+    return [this.flagged && `flagged-${this.flagged}`];
```

Comment:
Good question, actually. But we don't use that prefix elsewhere (e.g. FlagComponent, CardComponent among others) so for now it should be aligned with those (applying just the flagged value, i.e. `success`, `warning` etc)